ajax/libs/ppfish/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "ppfish",
+  "filename": "dist/ppfish.min.js",
```

Comment:
`ppfish.min.js` is the one should be filled with the main filename with its relative path to `basePath` in auto-update config. Therefore, the versions were added should be put `ajax/libs/ppfish/<version>/<filename>` to pass CI test.